packages/contracts/contracts/base/SemaphoreVerifierKeyPts.sol

```diff
@@ -0,0 +1,989 @@
+        hex"2373925e13c17f70c80783a3664e984c082f251bbe3c4698dff5daf3c70e5fdf"
+        hex"14c66f792e18c3133a0ec195583065986f5d8f297f5750ed78aee7b7fb6efaa1"
+        hex"1546abb4feb014cf7206953bcb52e5f16ac8d232723683674bc02854efa9c811"
+        hex"2dd9d7b31522a6b7fb653afd18e987a926fbd5377b5b0e415cf2614bdf0c2635"
+        hex"0102b6d8a26892985e76fd744ab01d06b524f57d634fc0b69a5d91355398d6ac"
+        hex"1ec5b7588982ece0b103edde74a6f327e6ccf694bc534195fe0effc7483f13a1"
+        hex"2b278acb099ad84fbc96b27d36269ba06a660e2bc9093059a2acb1998cf3f1fe"
+        hex"129253e64f00669a4cf7480180edd5c242544581499b9a233f0ce088acf1e4db"
+        hex"14f35ce5d343461157dbe0adb07355d95d2cd5aa6dba222b6a702af1e325d2bc"
+        hex"2d106ec5d71809dff7aec68762661bdfd5dbfe456cba5a7c150c73da2510f752"
+        hex"16ed7e6f631af32c7e9c7884b0ab0cf71a834ddc4c825748eb5d35451d809240"
+        hex"2dd68235d74c633a111bd167dec9a6371b137ae89530106a928c87743b38be3d"
+        hex"0076cc22cae0c2544d7e6d49cb00182253dab5dfcf6cbdb378f56b0714b9d4bf"
+        hex"2af75f56bee1cdbf00f1a0af44ee6ca3d7f8b3b42d38f87bf1da869240dc6477"
+        hex"092f4b09a0af31220825e29d275eeb08977e8d3a6a934cc16a6d562b2eca001a"
+        hex"04cb8d5ffe9fe85b21a785e261721288e9b5b75b400545a31d0d53270608b104"
+        hex"08287f21c644121d27ec9aa9f50ccbce17e7ebe37a9858da78ca1d207bd9d637"
+        hex"08d64375d1c8b872172ed0e50612b60c4b049e807f23d6fa8f4e61e06d52047a"
+        hex"034d11c3fb63ed48fc75af072a1084b3d0982b796c9dbb70c295e4c938688dd6"
+        hex"03457e18852f8480e869f8aa0665890d85a78c5458b77f682cbde6b7954b55d8"
+        hex"2cdc590f35d097cd61fc65b0dbaac9cd05916c807ec0e318de09c55d6facb637"
+        hex"0f2b077d67767da3df753109cb25329348253b3bb8a474b8f2383d269e7af7d3"
+        hex"1188362ee1c47b51ae57ee87cdc133d212b205f43ad871011e82684b3251a168"
+        hex"17b80922192c7ef6e95042838827e749bc017d1a96c6e69f4d945f6a40c80678"
+        hex"09380df1270bf5638a0dd4f2c7b7af4a5798b4280a4c98abd97c99272b34488d"
+        hex"055c100b6e332bb5d2bf6ba8f3f5945bd877926df3eaab055c5d42be090db0bf"
+        hex"02cc36716e8608d8ddd908e019707e77d50ecaae8f83a61fac66303fe0d98038"
+        hex"1913d591367d4749326ea822b20c1f015974fcd5ec6d659aedeac7b56f6fb789"
+        hex"1706e619141f6b4a93840023bb2342e759da6438e07ed974a5f1e3d506144c96"
+        hex"0bbe067b718cb861fe927716cba85cfc466eed0e9d2c1067bc7f9e356bc5d23c"
+        hex"1094541933220c2bef5ec7afe91782ec245208d58f93a08ec25eb59710a73623"
+        hex"03daf04d7a256119a10ba5f6653564dff54f8eb78577360ac477abab5ce87bf7"
+        hex"0cce5000e4686fab8ca621f1c65dda94eb8b5892b1aff79a9a5eb0a2e65efd53"
+        hex"14e0969768884cab782cafd4b81555a818559c4835fb6821ec46fd08ea418ac6"
+        hex"1ff343732becbc72450b2faedef7112b951edf2ee03bebe4b71365755e6d5518"
+        hex"10e06fbbc2176bff93433750bcbfec05aea120f193a1e4c5bf5993e098916f96";
+
+    function get_pts(uint256 merkleTreeDepth) internal pure returns (uint256[SET_SIZE] memory pts) {
+        bytes memory pt_bytes = VK_POINT_BYTES;
```

Comment:
Variable names should also be camelCase. Constants only should be snake_case (uppercase).

Reply:
updated